tests/projects/external-kit/build/rpms/core-kit/kit-metadata.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Why are we checking something in to the build directory of the project? This is problematic. The entire build directory should be gitignored and during a clean these directories are deleted and re-created.